app/views/components/ticket-panel/index.html

```diff
@@ -36,21 +36,16 @@ <h1 class="govuk-heading-xl">
           'aria-label': 'Sub navigation 2'
         },
         items: [{
-          html: ' <h2 class="govuk-heading-m govuk-!-margin-bottom-2">This is a heading 2</h2>
-          <p class="govuk-body">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-          <a class="govuk-button govuk-!-margin-bottom-1" data-module="govuk-button">
-            Save and continue
-          </a>',
+          html: ' <h2 class="govuk-heading-m govuk-!-margin-bottom-2"><a href="#" class="govuk-link">This is a heading 2</a></h2>
```

Comment:
@DanGuyMOJ are you able to take a look at this question from Greg?

Reply:
Hi @gregtyler. The link in header isn't (or at least shouldn't be) intrinsic to the component. I put that on there for the purposes or a CRIT it did where I just mocked it up to look like cards that link somewhere. That should be up to the designer/developer as to how they with to use the component. This component is essentially a container so I'm not wanting to focus too much on what it contains. However, having the panels link somewhere is something we might want to account for, in which case, the whole panel should be clickable like the NHS example here: https://service-manual.nhs.uk/design-system/components/card

Reply:
@johnnolan Would you be happy to change the examples back to the old button approach? Building on what Dan said, I'd rather keep our examples as standard as possible so that people use the container aspect rather than treating it as a full "card" component.

Reply:
@gregtyler Updated that now, thanks for the feedback both!